src/utils/ValidComponentChildren.js

```diff
@@ -137,11 +135,26 @@ function some(children, func, context) {
   return result;
 }
 
+function toArray(children) {
```

Comment:
Yeah I don't know, toArray does some bit of logic, but we may already have it covered